lib/StoryEngine/DataLayer/model/GlobalRuleDataModel.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+
+namespace StoryEngine.StoryEngineDataModel;
+
+public enum HowManyDataModel { all, any }
+public enum ListTypeForNodesDataModel { tags, storyElements, nodeIDs }
+public enum NodeStateDataModel { seen, notSeen }
```

Comment:
Could this be "seenState" with an enum of true/false? Could it be a boolean?

Reply:
Yeah, I don't see why not. I imagine I kept them all as enums so that each piece of the global rule would be set similarly (there may have been other reasons related to the old XML format too that made this make sense).

I could change `NodeStateDataModel` and `TagStateDataModel` to both be booleans but the other two have more choices so they have to stay enums. Do you think it's better to have all be enums for consistency or to have two be booleans?